### DISCUSSION TITLE
Stale Branches

### DISCUSSION THREAD

**Discussion by objecttothis** [opensourcepos/opensourcepos#3670] [Team Posts]
@jekkos @SteveIreland there are several stale branches in the repository.  Can any of them be deleted?

**Comment by objecttothis** [comment]
For example, the Release 2.2.2 branch.

**Comment by jekkos** [comment]
Feel free to clean that up. I removed already a couple some time ago.

**Comment by objecttothis** [comment]
@jekkos What about the gh-pages branch?  Since @WebShells went live with opensourcepos.org can we delete this or is it still useful?

**Comment by jekkos** [comment]
It's fine to get rid of it, basically it only contained a copy of the readme as html, so makes no sense to keep it .

**Comment by SteveIreland** [comment]
Regarding the multi-currency branch I suppose I'm still waiting on @jekkos to make the final decision.  There are a couple of users that have requested that it not be deleted (otherwise I would have deleted it myself) and that it be incorporated.  However, for that to happen I'm sure I'd need to bring it up to to the latest version.   I'm okay with whatever @jekkos decides.

**Comment by objecttothis** [comment]
@SteveIreland I just read through the conversation on #2399 and a few things are clear:
- There is demand for the enhancement.
- Some rework is needed on the enhancement design end to minimize the impact of the changes.
- That rework is outside the scope of what you can put into right now.

If it were me I wouldn't delete the branch and one of two things can happen:
1- @jekkos can stub out the design part and @SteveIreland or another dev can fill in the business logic.
2- Another dev can pick it up or be hired to complete the task by those that are interested in the feature.

I'll post this comment in #2399 in case others want to offer to pay for the feature.
